plugins/plugin-maven/che-plugin-maven-server/src/main/java/org/eclipse/che/plugin/maven/server/projecttype/handler/MavenProjectInitHandler.java

```diff
@@ -11,43 +11,25 @@
  */
 package org.eclipse.che.plugin.maven.server.projecttype.handler;
 
-import static org.eclipse.che.api.languageserver.LanguageServiceUtils.prefixURI;
 import static org.eclipse.che.plugin.maven.shared.MavenAttributes.MAVEN_ID;
 
 import com.google.inject.Inject;
 import com.google.inject.Singleton;
-import java.util.Collections;
-import org.eclipse.che.api.core.ConflictException;
-import org.eclipse.che.api.core.ForbiddenException;
-import org.eclipse.che.api.core.NotFoundException;
-import org.eclipse.che.api.core.ServerException;
 import org.eclipse.che.api.project.server.handlers.ProjectInitHandler;
-import org.eclipse.che.jdt.ls.extension.api.dto.UpdateWorkspaceParameters;
 import org.eclipse.che.plugin.java.languageserver.WorkspaceSynchronizer;
 
 /** @author Vitaly Parfonov */
 @Singleton
 public class MavenProjectInitHandler implements ProjectInitHandler {
 
-  private final WorkspaceSynchronizer workspaceSynchronizer;
-
   @Inject
-  public MavenProjectInitHandler(WorkspaceSynchronizer workspaceSynchronizer) {
-    this.workspaceSynchronizer = workspaceSynchronizer;
-  }
+  public MavenProjectInitHandler(WorkspaceSynchronizer workspaceSynchronizer) {}
 
   @Override
   public String getProjectType() {
     return MAVEN_ID;
   }
 
   @Override
-  public void onProjectInitialized(String projectFolder)
-      throws ServerException, ForbiddenException, ConflictException, NotFoundException {
-
-    UpdateWorkspaceParameters params = new UpdateWorkspaceParameters();
-    params.setAddedProjectsUri(Collections.singletonList(prefixURI(projectFolder)));
-
-    workspaceSynchronizer.syncronizerWorkspaceAsync(params);
-  }
+  public void onProjectInitialized(String projectFolder) {}
```

Comment:
This looks like we're not using this project initializer anymore. Can you explain how this made a selenium test fail and why we need to change it?
Generally, I think this kind of "meaningful" change would warrant a separate bug report and PR.

Reply:
Using `MavenProjectInitHandler` leads us to endless cycle:
1. Project with problems is imported
2. `JavaLanguageServerExtensionService.updateWorkspace` is invoked
3. `JavaLanguageServerExtensionService.updateProjectsWithProblems` is invoked
4. `MavenProjectInitHandler.onProjectInitialized` is invoked.
5. `WorkspaceSynchronizer.syncronizerWorkspaceAsync` is invoked
6. -> 1